server/tests/test_api.py

```diff
@@ -13,6 +13,24 @@ def generate_client_id():
     return str(uuid1())
 
 
+def set_client_id_cookie(client, id):
```

Comment:
We can create `server/tests/util.py` file and put those things there. I think, we will need it in some other places. Or you can put it in out `BaseTestCase`. Or leave it here for now :)